autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -1173,3 +1173,330 @@ def run(
                     logger.debug(f"Retry prompt: {prompt}")
 
         logger.debug("AIListGeneratorBlock.run completed")
+
+
+class Image(BaseModel):
+    content_type: str  # MIME type of the image
```

Comment:
base64 URI is already within this format:

`data:mime/type;base64=.......`
https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data#syntax

Reply:
https://github.com/Significant-Gravitas/AutoGPT/pull/9320 already uses this format, and this can be shown to the frontend directly as embedded image/video/audio

Reply:
would that type just be string?